spec/models/like_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject do
+    @user = User.new(name: 'Jack', photo: 'Tom.png', bio: 'bio')
+    Post.new(title: 'RoR', text: 'Hello', likes_counter: 1, author: @user, id: 4)
+  end
+
+  before { subject.save }
+
+  it 'should return the number of likes' do
+    subject.likes.new(author: @user, post: subject)
+    expect(subject.likes_counter).to eql 1
+  end
```

Comment:
- [ ] I can see you're testing the `likes_counter` attribute directly but remember that you have a method `update_likes_counter` in your Like model that returns the number of likes for a post, kindly test that method instead of testing the `attribute` itself because is required to test all the custom methods in each model.

Reply:
- [ ] please kindly add a test for the `update_likes_counter` as suggested by the previous reviewer. this method is not a private method so you should be able to create a test for it.